src/main/java/step2/Operator.java

```diff
@@ -0,0 +1,40 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+
+    PLUS("+", (num1, num2) -> num1 + num2),
+    MINUS("-", (num1, num2) -> num1 - num2),
+    MUL("*", (num1, num2) -> num1 * num2),
+    DIV("/", (num1, num2) -> num1 / num2);
+
+    private final String op;
+    private final BiFunction<Integer, Integer, Integer> expression;
+
+    Operator (String op, BiFunction<Integer, Integer, Integer> expression) {
+        this.op = op;
+        this.expression = expression;
+    }
+
+    public Operator getOperator (String input) {
+        return Arrays.stream(values())
+                .filter(op -> op.getOperator().equals(input))
+                .findFirst()
+                .orElseThrow(IllegalArgumentException::new);
+    }
```

Comment:
```suggestion
    public static Operator getOperator (String input) {
        return Optional.ofNullable(valuesMap.get(input))
                    .orElseThrow(() -> new IllegalArgumentException("기호가 다릅니다"));
    }
```
static 멤버 변수로 Map<String, Operator>를 갖게 작성해둔 덕에 연산 기호 String에 알맞는 Operator를 찾는 메서드는 이렇게 짧아 질 수 있었습니다.
그럼 이 메서드를 사용해야겠죠?